documentation/source/newsfragments/1457.feature.rst

```diff
@@ -0,0 +1,3 @@
+The name of the entry point symbol for libraries in :envvar:`GPI_EXTRA` can now be customized.
+The delimiter between each library in the list has changed from ``:`` to ``,``.
+It follows convention used by some simulators to load libraries.
```

Comment:
```suggestion
```
This line doesn't need to be in the release notes, lets leave details to the docs as in my suggestion above.